packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-answer.tsx

```diff
@@ -22,7 +24,23 @@ export class AtomicSmartSnippetAnswer {
     this.answerRendered.emit({height: this.wrapperElement!.scrollHeight});
   }
 
-  public render() {
+  private get sanitizedStyle() {
```

Comment:
I'm not sure this should apply here just like the result templates style tags. A customer could put any <script> tag anywhere in the page.

On the other end, what if it removes some valuable css ?

Reply:
> I'm not sure this should apply here just like the result templates style tags.

My worry is mostly somebody managing to escape out of the `style` tag. I'm not sure if that's possible in the first place, I couldn't find much information online, but I see that as being especially unlikely for result template style tags since they are already part of results, which are a more "obvious" thing to proof against XSS.

> A customer could put any <script> tag anywhere in the page.

my concern is mostly string interpolation / passing values through from a user, similar to storybook.